### PR TITLE
Chat: Register component

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/js/__internal/ui/chat/chat.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    $(this.element()).addClass(CHAT_CLASS);
+
+    super._initMarkup();
+  }
+
+  _optionChanged(args: Record<string, unknown>): void {
+    const { name } = args;
+
+    switch (name) {
+      case 'onMessageSend':
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/chat/chat.ts:37]: Why is the handling of 'items' skipped?

File: packages/devextreme/testing/tests/DevExpress.ui/defaultOptions.tests.js [hunk 2]
```diff
@@ -1332,6 +1333,13 @@ testComponentDefaults(ColorBox,
     }
 );
 
+testComponentDefaults(Chat,
+    {},
+    {
+        onMessageSend: null,
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui/defaultOptions.tests.js:1339]: What about default value of items option?

Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui/defaultOptions.tests.js:1339]: The option should not be processed here

File: packages/devextreme/js/__internal/ui/chat/chat.ts (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/js/__internal/ui/chat/chat.ts:37]: [7dc167e5df3ad2f72b8a6ba75100181eb5b847bb]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by iBat — APPROVED** [review]

**Pull request merged**